.github/workflows/verify.yml

```diff
@@ -72,6 +78,16 @@ jobs:
         with:
           path: .cache/webpack
           key: ${{ runner.os }}-build-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-build-
+      - name: Cache Jest
+        uses: actions/cache@v2
+        with:
+          path: .cache/jest
+          key: ${{ runner.os }}-jest-${{ hashFiles('package-lock.json') }}-${{ steps.get-date.outputs.date }}
```

Comment:
Why do we use the date as part of the cache key here?

Reply:
The idea of using the date was to keep the cache only valid for a limited amount of time - otherwise the cache is only replaced if the project dependencies are updated, even if the tests or source is updated. Judging from [the `restore-keys` docs](https://docs.github.com/en/actions/guides/caching-dependencies-to-speed-up-workflows#matching-a-cache-key) I think your suggestion would have approximately the same effect, the only real difference being the number of caches that will be created.

So, I think we should just go ahead and use `${{ github.run_number }}` instead. Thanks for the suggestion @developStorm!